framework/lib/test_generation/bin/randoopBloodhound.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
```

Comment:
There is a lot of cut-and-paste in this script.  Could we please add a command-line variable or environment variable to randoop.sh, rather than creating a script that differs from it in only a few lines?